### ISSUE TITLE
nix-prefetch-zip not found in nix-prefetch-scripts on macos

### ISSUE THREAD

**Issue by mckeankylej** [NixOS/nixpkgs#23230]
## Issue description
When I install nix-prefetch-scripts on macos nix-prefetch-zip doesnt end up on my path but it does end up in the nix store.

### Steps to reproduce
```bash
nix-env -i nix-prefetch-zip
```
Attempt to run `nix-prefetch-zip`

## Technical details

* System: (macos)
* Nix version: nix-env (Nix) 1.11.7
* Nixpkgs version: (17.03pre101919.a1431f3)

**Comment by Mic92** [comment]
Looks like: 97fc806a7b40babe7872bdb5cb1361bd7f8c2b3f deprecated it recently. But we probably should also mentioned that in the changelog for 17.03.

**Comment by copumpkin** [comment]
Keep in mind that 17.03 is a NixOS release, and that the macOS channel doesn't technically have releases, even though it comes off the same code.

**Comment by mckeankylej** [comment]
@Mic92 The problem is cabal2nix has not caught up, when I try to run cabal2nix I get this error:
```bash
bash-3.2$ cabal2nix https://github.com/mckeankylej/chronos.git > chronos.nix
cabal2nix: nix-prefetch-zip: createProcess: runInteractiveProcess: exec: does not exist (No such file or directory)
```
Is there some way I could get nix-prefetch-zip in the mean time?

**Comment by Mic92** [comment]
@mckeankylej you could use nixpkgs from git and revert this commit: `git revert 97fc806` Or you could just copy and paste the script from the commit to `~/bin/nix-prefetch-zip`. And then do `chmod +x ~/bin/nix-prefetch-zip` and add ~/bin to your PATH.

**Comment by mckeankylej** [comment]
Should I open this issue up on cabal2nix?

**Comment by Mic92** [comment]
@mckeankylej would be helpful

**Comment by globin** [comment]
Fixed in master and 17.03
